server/capabilities/gke_service_accounts_endpoint.go

```diff
@@ -0,0 +1,92 @@
+package capabilities
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"google.golang.org/api/iam/v1"
+
+	"net/http"
```

Comment:
Move this import to be next to the 3 other golang imports.

Reply:
Updated!